app/controllers/main/routes.py

```diff
@@ -42,9 +42,9 @@ def events(selectedTerm):
     participantRSVP = EventRsvp.select().where(EventRsvp.user == g.current_user)
     rsvpedEventsID = [event.event.id for event in participantRSVP]
     term = Term.get_by_id(currentTerm)
-    studentLedProgram = getStudentLedProgram(term)
-    trainingProgram = getTrainingProgram(term)
-    bonnerProgram = getBonnerProgram(term)
+    studentLedProgram = getStudentLedEvents(term)
+    trainingProgram = getTrainingEvents(term)
+    bonnerProgram = getBonnerEvents(term)
```

Comment:
Let's change these variables here and in the html template as well, to indicate we're talking about events and not a program